docs/howto/openstack/nova/new-vgpu-server.md

```diff
@@ -5,70 +5,98 @@ In select
 and
 [{{brand_compliant}}](../../../reference/features/compliant.md#virtualization)
 regions, you can instantiate virtual machines --- or simply _servers_
---- with virtual GPU (vGPU) support. For that, you use the `openstack`
-CLI utility.
+--- with virtual GPU (vGPU) support. For that, you may either use the
+{{brand}} Dashboard or the `openstack` CLI utility.
```

Comment:
Please don't invent another term for CCMP.

```suggestion
{{gui}} or the `openstack` CLI utility.
```

Reply:
No, no new term.